get-started/first-steps-vs-code.md

```diff
@@ -21,13 +21,27 @@ In this tutorial, you will enhance an existing .NET MAUI application by adding a
 
 @[template](/_contentTemplates/get-started.md#start-free-trial)
 
-## Step 1: Set Up Your .NET MAUI Project
+## Step 1: Downnload Telerik MAUI extension for Visaul Studio Code
 
-In this step, you will create a basic .NET MAUI project as a starting point for your application development, and then run it:
+To use the **Telerik UI for .NET MAUI Template Wizard**, install the `Telerik UI for .NET MAUI Productivity Tools` extension. You can get the extension from:
 
-1. Open Visual Studio Code and select `Cmd/Ctrl+Shift+P`. Enter **.NET: New Project...** in the input field.
+* <a href="https://marketplace.visualstudio.com/items?itemName=TelerikInc.telerik-maui-productivity-tools" target="_blank">The Visual Studio Code Marketplace.</a>
 
-1. Select the **.NET MAUI App** option.
+## Step 2: Start the Telerik MAUI Wizard
+
+You can start the Telerik UI for .NET MAUI Template Wizard in either of the following ways:
+
+* Click the **Telerik Extensions Information** icon in the Visual Studio Code activity bar.
+
+    1. Click on the **Telerik Extensions Information** icon
+    1. Select **MAUI** tab
+    1. Click **Create New Project**.
+    1. The **Telerik UI for .NET MAUI Template Wizard** opens
+    1. Using the Template Wizard you can create a new **Blank App** or **Blank Shell App**.
```

Comment:
The next set of instructions is related to the configuration of the new project and can be presented as a separate list.
```suggestion
```